announcements/1.16.0.md

```diff
@@ -0,0 +1,100 @@
+* TURN credentials for the container are now dynamically allocated and are no longer created with a constant lifetime.
+
+### Data channel v2
+
+[Version 2 of the OOB data exchange protocol](https://anbox-cloud.io/docs/howto/stream/oob-data#oob-v2), which Anbox Cloud introduced in version 1.15, provides a full-duplex bidirectional data transmission between the Android container and the WebRTC client. In the 1.16 release, Anbox Cloud introduced the Anbox WebRTC data proxy service as an extension to version 2 of the OOB data. The data proxy service exposes a binder service in the Android container that can be used by an Android application to communicate with a WebRTC client over the Anbox WebRTC platform.
+
+### Ubuntu 22.04 support
+
+The Juju charms of Anbox Cloud now fully support Ubuntu 22.04. The Anbox Cloud Appliance can run on Ubuntu 22.04 already since the 1.15 release.
+
+By default, the Anbox Cloud bundles ([`anbox-cloud`](https://charmhub.io/anbox-cloud), [`anbox-cloud-core`](https://charmhub.io/anbox-cloud-core)) available from [Charmhub](https://charmhub.io/) use Ubuntu 22.04.
+
+With the introduction of Ubuntu 22.04, support for Ubuntu 18.04 was removed.
+
+### AMS improvements
+
+#### APT mirror configuration
+
+The APT mirror used inside Anbox containers has been hard-coded to `archive.ubuntu.com` or `ports.ubuntu.com`, which is not always ideal. To allow for easier modification, AMS now provides a `container.apt_mirror` configuration option. This option can be set to a custom Ubuntu mirror at runtime and will then be used by all containers that are started afterwards.
+
+#### Networking ACLs
+
+A new AMS charm configuration `use_network_acl` has been added. This configuration enables AMS to use [LXD network ACLs](https://linuxcontainers.org/lxd/docs/master/howto/network_acls/) <!-- wokeignore:rule=master --> rather than the AMS node controller (which is used by default) to restrict the network access of Anbox containers. The option is experimental and not ready for general use, but will in the future replace the functionality that is currently part of the AMS node controller.
+
+### Other
+
+* Included Android security updates for November 2022 (see [Android Security Bulletin - November 2022](https://source.android.com/docs/security/bulletin/2022-11-01) for more information).
+* Updated Android WebView to [107.0.5304.105](https://chromereleases.googleblog.com/2022/11/chrome-for-android-update.html).
+* The used NVIDIA driver series is updated to [515](https://docs.nvidia.com/datacenter/tesla/index.html) which is supported until May 2023.
+* The appliance now deploy Ubuntu 22.04 based LXD containers on new deployments. Existing deployments will keep their existing Ubuntu 20.04 ones.
+* AMD GPU support is now automatically detected by the [`ams-lxd`](https://charmhub.io/ams-lxd) Juju charm and the Anbox Cloud Appliance and the kernel `amdgpu` driver configured for best performance and support.
```

Comment:
```suggestion
* Updated the [`ams-lxd`](https://charmhub.io/ams-lxd) Juju charm and the Anbox Cloud Appliance to automatically detect AMD GPU support and configure the kernel `amdgpu` driver for best performance and support.
```
Is that correct? The sentence was quite ambiguous.

Reply:
Yes, after your change it's correct.